src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RSAOAEPKeyExchangeDeformatter.cs

```diff
@@ -0,0 +1,47 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Cryptography
+{
+    public class RSAOAEPKeyExchangeDeformatter : AsymmetricKeyExchangeDeformatter
+    {
+        private RSA _rsaKey;
+
+        public RSAOAEPKeyExchangeDeformatter() { }
+        public RSAOAEPKeyExchangeDeformatter(AsymmetricAlgorithm key)
+        {
+            if (key == null)
+                throw new ArgumentNullException(nameof(key));
+
+            _rsaKey = (RSA)key;
+        }
+
+        public override string Parameters
+        {
+            get
```

Comment:
You thumbs-up'd this, but didn't make the corresponding change.  I'm confused...